README.md

```diff
@@ -34,6 +30,23 @@ the ARTwarp Wiki: https://github.com/dolphin-acoustics-vip/artwarp/wiki
 
 - Wiki: https://github.com/dolphin-acoustics-vip/artwarp/wiki
 
+## Getting Started
+
+1. Ensure you have a compatible version of MATLAB. ARTwarp works well with most
+recent versions of MATLAB, but is recommended to run on versions no older than
+2022
+
+2. Download the source code from one of the releases at the releases page here:
+https://github.com/dolphin-acoustics-vip/artwarp/releases. Download it to a
+pathway that can be accessed through MATLAB. There is a changes log available
+which lists new functionality and bug fixes added in each release:
+https://github.com/dolphin-acoustics-vip/artwarp/blob/stable-1/CHANGES.md
```

Comment:
Of course the file will be included in their downloaded version - OTOH, ok to give URL, so they may see what's coming too, so fine.

Reply:
yes, so keep the URL.